HopsanGUI/Widgets/TextEditorWidget.cpp

```diff
@@ -62,6 +62,8 @@ TextEditorWidget::TextEditorWidget(QFileInfo scriptFileInfo, HighlighterTypeEnum
     QFont font("Monospace");
     font.setStyleHint(QFont::TypeWriter);
     mpEditor->setFont(font);
+    QFontMetrics metrics(font);
+    mpEditor->setTabStopWidth(4*metrics.width(' '));
```

Comment:
Maybe tab width should be configurable (no need to fix now) also I have started to prefer two spaces

Reply:
We could add that later if people request it. This is only for loading existing files anyway. The default Qt behavior was to show tabs as ~10 characters, which looked ridiculous.